test/e2e_node/gci-init.sh

```diff
@@ -26,7 +26,11 @@ fi
 
 if [ "${CONTAINERD_CGROUPV2:-"false"}"  == "true" ]; then
   # check cos image
-  if uname -a | grep -q cos; then
+  OS_ID=""
+  if [ -r /etc/os-release ]; then
+    OS_ID="$(. /etc/os-release && echo "$ID")"
+  fi
+  if [[ "${OS_ID}" == "cos" ]]; then
```

Comment:
nit: perhaps avoid using bashisms here (to allow the script to be functional on standard POSIX (`/bin/sh`) shells;

```suggestion
  if [ "${OS_ID}" = "cos" ]; then
```

Reply:
Thanks, updated.